src/docs/deployment/obfuscate.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Might be worth noting some of the caveats, code like `expect(Foo.runtimeTime.toString(), equals('Foo'))` will fail because the Foo type was renamed.